### ISSUE TITLE
Make larger `batch_size` available when training with GPU

### ISSUE THREAD

**Issue by pinskyrobin** [pinskyrobin/FGMCD#2]
Now, one can only train this model with `batch_size=8` or even smaller on CUDA.

Desirable `batch_size` would be greater than `64`.

**Comment by pinskyrobin** [comment]
partly solved by using gradient accumulation
